common/app/views/support/Profile.scala

```diff
@@ -44,7 +44,7 @@ sealed trait ElementProfile {
     bestFor(image).flatMap(_.altText)
 
   // NOTE - if you modify this in any way there is a decent chance that you decache all our images :(
-  val qualityparam = if (hidpi) {"q=20"} else {"q=55"}
+  val qualityparam = if (hidpi) {"quality=45"} else {"quality=85"}
```

Comment:
you need to update the `ShareImage` quality param - the `h` and `w` params there need fixing too.

Reply:
Done.